lib/Carton/Doc/FAQ.pod

```diff
@@ -109,4 +109,12 @@ L<Any::Moose>.
 
 =back
 
+=head2 I'm using perlbrew, but C<< carton install >> uses system perl, how to fix?
 
+Instead of C<< carton install >>, use C<< perl -S carton install >>.
+
+This will force the use of the C<perl> from your PATH, and not the C</usr/bin/perl>
+that is hard-coded at the top of the C<carton> script.
```

Comment:
I understood the explanation via IRC, and I understand that this was caused by a bad initial cpanm install on my system.

I'll rewrite that comment to me more clear.

See also my next commit. If you think this FAQ entries are not worthy, I can just drop them…